src/applications/public-outreach-materials/libraries/library-filters.js

```diff
@@ -31,25 +26,44 @@ export function libraryCount() {
         numCards < 0 ? 0 : numCards;
     }
     document.getElementById('total-all').innerText = ` of ${cards.length}`;
-    if (numCards < 1) {
-      document.getElementById('va-pager-div').style.display = 'none';
-      document.getElementById('no-results').style.display = 'block';
+    if (libraryGetQParam() === benefit) {
+      if (numCards < 1 && document.getElementById('no-results')) {
+        document.getElementById('va-pager-div').style.display = 'none';
+        document.getElementById('no-results').style.display = 'block';
+      }
     }
   }
 }
 
+export function libraryCurrent() {
+  cards.forEach(element => {
+    const numVal = element.getAttribute('data-number');
+    const itemsPerPage = 10;
+    if (
+      numVal > activePage * itemsPerPage ||
+      numVal < activePage * itemsPerPage - 9
```

Comment:
Alright, so I think I understand what this is doing. If it's outside of the bounds the active page is supposed to show, hide the card, otherwise show it.

Does the `data-number` attribute start at 1 or 0? If it starts at 0, I think the first card will always be hidden.

Aside from that, I'd like to nix that `- 9` as well since it's a derivative of `itemsPerPage`. What do you think of this?
```suggestion
      numVal <= (activePage - 1) * itemsPerPage
```

I had to write it out with real numbers to make sure it has the same functionality:
```
numVal = 10
activePage = 2

// Previously
10 < 2 * 10 - 9 // 10 < 11

// Modified
10 <= (2 - 1) * 10 // 10 <= 10 -- same range as above
```

Reply:
`data-number` starts at 1, @cvalarida - I just pushed the above suggestion.